test/fixtures/index.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
   inputWebPWithTransparency: getPath('5_webp_a.webp'), // http://www.gstatic.com/webp/gallery3/5_webp_a.webp
   inputTiff: getPath('G31D.TIF'), // http://www.fileformat.info/format/tiff/sample/e6c9a6e5253348f4aef6d17b534360ab/index.htm
   inputTiffCielab: getPath('cielab-dagams.tiff'), // https://github.com/lovell/sharp/issues/646
+  inputTiffUncompressed: getPath('uncompressed_tiff.tiff'),
```

Comment:
As long as you're happy to licence it, perhaps using [CC0](https://creativecommons.org/publicdomain/zero/1.0/), that's OK.

If possible, please could you use a smaller file, or perhaps only a portion of this file, as this is 15MB!